### ISSUE TITLE
feature: set feelimit percentage in settings

### ISSUE THREAD

**Issue by ottosuess** [LN-Zap/zap-iOS#278]
You can set "The fee limit expressed as a percentage of the payment amount" in the lnd api.

https://api.lightning.community/#feelimit

this could be an option in settings. basically the same thing as @chitowncrispy did in this pr: https://github.com/LN-Zap/zap-iOS/pull/274

idk what limits we should let the user choose. maybe:
1%
3%
5%
10%

any thoughts @JimmyMow ?

**Comment by maraph** [comment]
I've implemented this on Android: https://github.com/LN-Zap/zap-android/pull/122

So currently for payments below 100 sats this fee limits are ignored. (limit for them is 100%)
The default limit for payments above 100 sats is at 3%.

Do you think those values are okayish? Or would you go for other ones?
@ottosuess @JimmyMow

**Comment by ottosuess** [comment]
@maraph when do you show the alert in the android app? when `queryRoute` returns a fee higher than the % limit?

let's say the route returned by `queryRoute` is below the fee limit, but fails. and an other route with a fee above the fee limit is chosen. do we get an alert or does it just silently fail?

**Comment by maraph** [comment]
@ottosuess Yes, if the possible fee will be higher than the limit, we show the alert.

To the second questions right: If we set the limit and LND doesn't find a route within the fee limit, the payment will fail with "no path to destination".
